tools/locate_bins.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh -e
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+have_command () {
+  eval "$1 --version >> /dev/null 2>&1"
+  if [ $? -ne 0 ]
+  then
+    echo "false"
+  fi
+  echo "true"
+}
+
+eval "sudo -n echo >> /dev/null 2>&1"
```

Comment:
can you integrate it so that bootstrap.sh and this script would have it added the same?

Reply:
I don't know what you mean by this.

Reply:
Did you read how bootstrap.sh handles sude?

Reply:
I have checked bootstrap script and it simply uses `sudo` without 
* checking if `sudo` is installed
* checking if `sudo` is required for a particular command
* checking if you the right to run a program using `sudo`
* checking if `sudo` will run non-interactively (it will prompt you literally five times for the same password in the boostrap script)

The when making the command wrappers, the script checks if `sudo` is installed but **only** uses it when it is needed (current user lacks rights), allowed (you are permitted via sudoers), and non-interactive (so you don't get prompted 20 times per build).